pkg/sqlite/load.go

```diff
@@ -114,6 +116,14 @@ func (s *sqlLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error
 		return err
 	}
 	for img := range imgs {
+		if img == "" {
+			// This condition is something that exists in
+			// the wild and must be resolved by operator
+			// authors, so it's not currently considered
+			// fatal at catalog build time.
+			s.logger.Warnf("warning: operator %q specifies an empty image reference")
```

Comment:
Should we be more descriptive here and potentially point the user to the actual bundle path that specifies the empty image reference if this is being loaded from a bundle image?

Reply:
Isn't this also not providing a value for the `%q` field?